website/docs/amazon-emr-on-eks/emr-eks-karpenter.md

```diff
@@ -2,32 +2,158 @@
+
+```bash
+aws eks --region us-west-2 update-kubeconfig --name emr-eks-karpenter # Creates k8s config file to authenticate with EKS Cluster
+
+kubectl get nodes # Output shows the EKS Managed Node group nodes
+
+kubectl get ns | grep emr-data-team # Output shows emr-data-team-a and emr-data-team-b namespaces for data teams
+
+kubectl get pods --namespace=prometheus # Output shows Prometheus server and Node exporter pods
+
+kubectl get pods --namespace=vpa  # Output shows Vertical Pod Autoscaler pods
+
+kubectl get pods --namespace=kube-system | grep  metrics-server # Output shows Metric Server pod
 
-Additionally, all the **Karpenter Node templates** use **RAID0 configuration** to ensure Spark jobs can refer to `/local1` as one folder even if the instances have one or more SSD disks.
+kubectl get pods --namespace=kube-system | grep  cluster-autoscaler # Output shows Cluster Autoscaler pod
+```
 
-This pattern deploys three Karpenter provisioners, and it also provides guidance on using **Apache YuniKorn as a batch scheduler** to **gang-schedule** Spark jobs.
+## Run Sample Spark job
 
-This example showcases how multiple data teams within an organization can run Spark jobs using Karpenter provisioners that are unique to each workload.
-For example, you can use a compute-optimized provisioner that has taints and use pod templates to specify tolerations so that you can run Spark on compute-optimized EC2 instances.
+The pattern shows how to run spark jobs in a multi-tenant EKS cluster. The examples showcases two data teams using namespaces `emr-data-team-a` and `emr-data-team-b` mapped to their EMR virtual clusters. You can use different Karpenter provisioners for each team so that they can submit jobs that are unique to their workload. Teams can also use different storage requirements to run their Spark jobs. For example, you can use compute optimized provisioner that has `taints` and specify `tolerations` using pod templates so that you can run spark on compute optimized EC2 instances. In terms of storage, you can decide whether to use [EC2 instance-store](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html) or [EBS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html) or [FSx for lustre](https://docs.aws.amazon.com/fsx/latest/LustreGuide/what-is.html) volumes for data processing. The default storage that is used in these examples is EC2 instance store because of performance benefit
 
 - `spark-compute-optimized` provisioner to run spark jobs on `c5d` instances.
 - `spark-memory-optimized` provisioner to run spark jobs on `r5d` instances.
 - `spark-graviton-memory-optimized` provisioner to run spark jobs on `r6gd` Graviton instances(`ARM64`).
 
-Let's review the Karpenter provisioner for computed optimized instances deployed by this pattern.
+<Tabs>
+<TabItem value="spark-compute-optimized" lebal="spark-compute-optimized"default>
```

Comment:
I love this tab feature. Lets add the graviton example as well `spark-graviton-memory-optimized`

we need the following 4 sections

1. Run Sample Spark job with NVMe SSD  - This will have three tabs - one for each example
2. Run Sample Spark job tih FSx for Lustre - this will have two tabs - one for each example
3. Run Sample Spark job with EBS - this will have one tab - for one example
4. Apache YuniKorn with Gang scheduling one section

Reply:
Its doing this sort of now. Let me modify FSx for Lustre option and include it in the tab. We can add additional examples in the future if needed